Automattic/jetpack
# My Jetpack: do not enable Licensing UI if My Jetpack is disabled

Fixes #25666

#### Changes proposed in this Pull Request:

This is true for Multisite environments, for example. If My Jetpack isn't available, the licensing UI in My Jetpack should not be available either.

> **Note**
> Beyond this PR, this brings the question of whether the licensing UI should be shown on multisite environments in the Jetpack dashboard.
> This question is somewhat related to this discussion: p1659447314971209-slack-CS8UYNPEE
> @Automattic/jetpack-origin Would you have an opinion on this?

#### Other information:


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

1. Start with a site that's part of a Multisite network.
2. Connect your site to WordPress.com.
3. Go to https://cloud.jetpack.com/pricing and purchase a plan; do not attach it to any site yet.
4. Go back to your site's WP-Admin
5. Go to Jetpack > Dashboard > My Plan
6. Click on the link to activate a product
7. You should not be redirected to a My Jetpack page
